packages/core/src/components/Modal/__tests__/__snapshots__/Modal.snapshot.test.js.snap

```diff
@@ -154,7 +154,6 @@ Snapshot Diff:
 +       <div
 +         class="typography primary start text text1Normal container"
 +         data-testid="modal-content"
-+         style="--text-clamp-lines: 1;"
```

Comment:
seem like an unwanted change?

Reply:
I think it is ok - I've removed text-clamp if single ellipsis line